src/game.cpp

```diff
@@ -8542,12 +8542,12 @@ void game::zones_manager()
                                            pointPlayer.x,
                                            pointPlayer.y
                                           ),
-                              direction_name_short(direction_from(pCenter.x,
-                                                   pCenter.y,
-                                                   pointPlayer.x,
-                                                   pointPlayer.y
-                                                                 )
-                                                  ).c_str()
+                              direction_name(direction_from(pCenter.x,
```

Comment:
Fair enough - I debated whether to use the bool or keep the split; I'll revert to the split and just forward to a common implementation then.